docs/upgrade-guide/upgrade211-30.md

```diff
@@ -0,0 +1,167 @@
+
+
+**Upgrade to v11:**
+
+To update ANGULAR version from 10.0 to 11.0 by running the below update commands
+
+```sh
+ng update @angular/core@11 @angular/cli@11 –force
+npm i
+```      
+      
+Below are the some updates when you upgrade to v11.0
+
+    • ANGULAR now requires TypeScript 4.0. ng update will migrate you automatically.
+    
+    • Support for IE9, IE10, and IE mobile has been removed.
+
+
+
+**Upgrade to v12:**
+
+To update ANGULAR version from 11.0 to 12.0 by running the below update commands
+
+```sh
+ng update @angular/core@12 @angular/cli@12 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v12.0
+
+    • ANGULAR now requires TypeScript 4.2. ng update will update you automatically.
+    
+    • IE11 support has been deprecated.
+    
+    • You can no longer use ANGULAR with Node.js version 10 or older.
+    
+    • Update zone.js to version 0.11.4. ng update will update this dependency automatically.
+    
+    • Remove 'emitDecoratorMetadata' TypeScript compiler option.
+    
+    • Remove deprecated ViewEngine-based i18n build and extract options.
+    
+    • Updates Web Worker consumer usage to use the new syntax supported directly by Webpack 5.
+    
+    • `XhrFactory` has been moved from `@angular/common/http` to `@angular/common`.
+
+
+
+**Upgrade to v13:**
+
+To update ANGULAR version from 12.0 to 13.0 by running the below update commands
+
+```sh
+ng update @angular/core@13 @angular/cli@13 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v13.0
+
+    • ANGULAR now uses TypeScript 4.4.
+    
+    • Make sure you are using Node 12.20.0 or later
+    
+    • Use [routerLink]="" in templates to [routerLink]="[]" because these links are likely intended to route to the current page with updated fragment/query params.
+    
+    • As of ANGULAR version 13, `entryComponents` are no longer necessary.
+    
+    • Remove polyfills required only for Internet Explorer which is no longer supported.
+    
+    • Remove no longer valid ANGULAR schematic options from `angular.json`.
+
+
+
+**Upgrade to v14:**
+
+To update ANGULAR version from 13.0 to 14.0 by running the below update commands
+
+```sh
+ng update @angular/core@14 @angular/cli@14 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v14.0
+
+    • ANGULAR now uses TypeScript 4.6
+    
+    • Make sure you are using Node 14.15.0 or later
+    
+    • Update ANGULAR packages 'dependencies' and 'devDependencies' version prefix to '^' instead of '~'.
+    
+    • Remove 'package.json' files from library projects secondary entrypoints.
+    
+    • Update TypeScript compilation target to 'ES2020'.
+
+
+Make sure you have install dev dependency for the below dependecies
+
+inquirer, winston
```

Comment:
you don't need this line as you already give example what to install later in the code